introduction.md

```diff
@@ -1 +1,12 @@
 Kris works in kansas city somewhere. Javascript is my favorite right now.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?